Tests/FluentTests/ModelTests.swift

```diff
@@ -98,6 +98,63 @@ class ModelTests: XCTestCase {
         XCTAssert(test.id != nil)
     }
 
+    final class Box : Entity, Preparation {
```

Comment:
`Box` should go in the `Utilities/...` files if possible (to keep unit tests clean).

Reply:
Right, I didn't notice the pattern